packages/monaco/src/browser/monaco-outline-contribution.ts

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from "inversify";
+import SymbolInformation = monaco.modes.SymbolInformation;
+import SymbolKind = monaco.modes.SymbolKind;
+import { FrontendApplicationContribution, FrontendApplication, OpenerService, open, ITreeNode } from "@theia/core/lib/browser";
+import { MonacoTextModelService } from './monaco-text-model-service';
+import { Range, EditorManager, EditorWidget } from '@theia/editor/lib/browser';
+import DocumentSymbolProviderRegistry = monaco.modes.DocumentSymbolProviderRegistry;
+import CancellationTokenSource = monaco.cancellation.CancellationTokenSource;
+import { DisposableCollection, CommandContribution, CommandRegistry } from "@theia/core";
+import { OutlineViewManager, OutlineSymbolInformationNode } from "@theia/outline-view/lib/browser/outline-view-manager";
+import URI from "@theia/core/lib/common/uri";
+
+export namespace OUTLINE_COMMANDS {
```

Comment:
it should be something like `MONACO_OUTLINE_COMMANDS` and command should be prefixed with `monaco.`, otherwise it sounds as generic outline commands or maybe they are generic and should be declared in the outline extension

Reply:
they are specific to monaco so should be prefixed as you said